Update Get-CsOnlineUser.md (#9299)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsOnlineUser.md

### -------------------------- Example 5 --------------------------
```
Get-CsOnlineUser -Filter {LineURI -eq "tel:+1234"}
Get-CsOnlineUser -Filter {LineURI -eq "tel:+1234,ext:"}
Get-CsOnlineUser -Filter {LineURI -eq "1234"}
```

> Examples above returns information for user accounts that have been assigned a designated phone number. 

Reviewer: ```suggestion
Example 5 returns information for user accounts that have been assigned a designated phone number. 
```

- UserDirSyncEnabled
- TeamsWorkLoadPolicy
- VoiceRoutingPolicy

> - LastName (available after TPM 4.2.0)

Reviewer: ```suggestion
- LastName (available in Teams PowerShell Module 4.2.0 and later)
```

- All users with an AssignedPlan that starts with MCO: `-Filter "AssignedPlan -eq 'MCO*'"`
- All users with an AssignedPlan that contains MCO: `-Filter "AssignedPlan -eq '*MCO*'"`
- All users with an AssignedPlan that ends with "MCO": `-Filter "AssignedPlan -eq '*MCO'"`

**Policy Attributes**:

- PolicyProperty comparison works only when "Authority" is provided in the value. For ex: `-Filter "TeamsMessagingPolicy -eq '<Authority>:<Value>'"`
"Authority" can contain any of these two values: Host or Tenant for a policy type (configurations that are provided by default are referred to as Host configurations while admin-created configurations are considered Tenant configurations). The following are more examples:

- Filter "TeamsMessagingPolicy -eq 'Host:EduStudent'"
- Filter "TeamsMessagingPolicy -eq 'Tenant:TestDemoPolicy'"

- In the Teams PowerShell Module version 3.0.0 or later, the output format of Policies has now changed from String to JSON type UserPolicyDefinition.

*Change in Filter operators*:

The following filter syntaxes have been modified in Teams PowerShell Module 3.0.0 and later:

- -not, -lt, -gt: These operators have been dropped.

> - -ge:  These operators are not supported with policy properties.

Reviewer 2: ```suggestion
- -ge:  This operator is not supported with policy properties.
```

In the Teams PowerShell Module version 3.0.0 or later, the format of the AssignedPlan and ProvisionedPlan attributes has changed from XML to JSON array. Previous XML filters (For example, `-Filter "AssignedPlan -eq '<some-xml-string>'"`) will no longer work. Instead, you need to update your filters to use one of the following formats:

- All users with an AssignedPlan that starts with MCO: `-Filter "AssignedPlan -eq 'MCO*'"`
- All users with an AssignedPlan that contains MCO: `-Filter "AssignedPlan -eq '*MCO*'"`
- All users with an AssignedPlan that ends with "MCO": `-Filter "AssignedPlan -eq '*MCO'"`

> 

Reviewer: ```suggestion
```

**Policy Attributes**:

- PolicyProperty comparison works only when "Authority" is provided in the value. For ex: `-Filter "TeamsMessagingPolicy -eq '<Authority>:<Value>'"`

> "Authority" can contain any of these two values: Host or Tenant for a policy type (configurations that are provided by default are referred to as Host configurations while admin-created configurations are considered Tenant configurations). Below are more examples:

Reviewer: ```suggestion
"Authority" can contain any of these two values: Host or Tenant for a policy type (configurations that are provided by default are referred to as Host configurations while admin-created configurations are considered Tenant configurations). The following are more examples:
```

> 1. Filter "TeamsMessagingPolicy -eq 'Host:EduStudent'"
> 2. Filter "TeamsMessagingPolicy -eq 'Tenant:TestDemoPolicy'"

Reviewer: ```suggestion
- Filter "TeamsMessagingPolicy -eq 'Host:EduStudent'"
- Filter "TeamsMessagingPolicy -eq 'Tenant:TestDemoPolicy'"
```

- OnPremSamAccountName
- CallerIdPolicy
- Fax

> - LastName (available after TPM 4.2.0)

Reviewer: ```suggestion
- LastName (available in Teams PowerShell Module 4.2.0 and later)
```
